tensorflow/compiler/mlir/lite/utils/validators.cc

```diff
@@ -91,6 +91,20 @@ bool TFIntListIs1XYZ1(Operation *op, StringRef name, IntegerAttr *x,
   return true;
 }
 
+// Returns true if the attribute is an integer list of the form [1, X, Y, Z, 1],
+bool TFIntListIs1XYZ1(const Attribute attr) {
+  const auto &elements = attr.cast<ArrayAttr>().getValue();
+  if (elements.size() != 5 ||
+      std::any_of(elements.begin(), elements.end(),
+                  [](Attribute e) { return !e.isa<IntegerAttr>(); }))
```

Comment:
What can be it be except an IntegerAttr? I ask as in line 96 there is assumption that this is only used where ArrayAttr is already verified to be the attribute type and the function name says it should only be used for int lists. So seems like one assumption is checked while the other is assumed (and cast will just assert in debug mode).